qemu/tests/cpu_info_check.py

```diff
@@ -0,0 +1,134 @@
+                                          "avx512vl,clflushopt,avx512_vnni"),
+                   "Skylake-Server": "avx512f,clwb,xgetbv1,pcid",
+                   "Skylake-Client": "xgetbv1,pcid",
+                   "Broadwell": "adx,rdseed,3dnowprefetch,hle",
+                   "Broadwell-noTSX": "adx,rdseed,3dnowprefetch",
+                   "Haswell": "fma,avx2,movbe,hle",
+                   "Haswell-noTSX": "fma,avx2,movbe",
+                   "IvyBridge": "f16c,fsgsbase,erms",
+                   "SandyBridge": ("avx,xsave,aes,sse4_2|sse4.2,sse4.1|sse4_1,"
+                                   "cx16,ssse3"),
+                   "Westmere": "aes,sse4.2|sse4_2,sse4.1|sse4_1,cx16,ssse3",
+                   "Nehalem": "sse4.2|sse4_2,sse4.1|sse4_1,cx16,ssse3",
+                   "Penryn": "sse4.1|sse4_1,cx16,ssse3",
+                   "Conroe": "ssse3"}
+
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    match = re.search(r'[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?',
+                      qemu_version)
+    host_qemu = match.group(0)
+    if host_qemu in VersionInterval('[3.1.0,)'):
+        cpu_types.extend(['Cascadelake-Server', 'Icelake-Client',
+                          'Icelake-Server', 'KnightsMill'])
+    if (host_qemu in VersionInterval('[4.2.0, )') and
+            params.get('os_variant') == 'rhel8'):
+        cpu_types.extend(
+            ['Cascadelake-Server-noTSX', 'Icelake-Client-noTSX',
+             'Icelake-Server-noTSX', 'Cascadelake-Server-v1',
+             'Cascadelake-Server-v2', 'Cascadelake-Server-v3',
+             'Icelake-Server-v1', 'Icelake-Server-v2', 'EPYC-v1', 'EPYC-v2'])
+        if params.get('os_app_variant') == '1':
+            cpu_types.extend(['Cooperlake', 'Cooperlake-v1', 'Snowridge',
+                              'Snowridge-v1', 'Snowridge-v2'])
+    if host_qemu in VersionInterval('[5.0.0, )'):
+        cpu_types.extend(['EPYC-Rome', 'EPYC-Rome-v1', 'EPYC-v3'])
+
+    qemu_binary = utils_misc.get_qemu_binary(params)
+    query_cmd = "%s -cpu ? | awk '{print $2}'" % qemu_binary
+    qemu_binary_output = process.system_output(
+        query_cmd, shell=True).decode().splitlines()
+    cpuid_index = qemu_binary_output.index('CPUID')
+    cpu_models_binary = qemu_binary_output[1: cpuid_index - 1]
+    cpu_flags_binary = qemu_binary_output[cpuid_index + 1:]
+    params['start_vm'] = 'yes'
+    vm_name = params['main_vm']
+    env_process.preprocess_vm(test, params, env, vm_name)
+    vm = env.get_vm(vm_name)
+    # query cpu model supported by qemu
+    qmp_model_output = str(vm.monitor.cmd('qom-list-types'))
+    qmp_def_output = str(vm.monitor.cmd('query-cpu-definitions'))
+
+    # Check if all the output contain expected cpu models
+    missing = [[], [], []]
+    output_list = [cpu_models_binary, qmp_model_output, qmp_def_output]
+    for cpu_model in cpu_types:
```

Comment:
How about print some log to indicate the steps?

Reply:
@huangyum Okay I will, thanks.